.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Test on Comment"
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: |
+        echo "$GITHUB_CONTEXT"
+    - name: Get PR branch
+      uses: xt0rted/pull-request-comment-branch@v1
+      id: comment-branch 
+    - name: Set latest commit status as pending
+      uses: myrotvorets/set-commit-status-action@master
+      with:
+        sha: ${{ steps.comment-branch.outputs.head_sha }}
+        token: ${{ secrets.CI_TOKEN }}
+        status: pending
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-west-2
+    - name: Run CodeBuild
+      uses: aws-actions/aws-codebuild-run-build@v1
+      if: github.event.issue.pull_request && contains(github.event.comment.body, '/do-e2e-test')   # check the comment if it contains the keywords
+      with:
+        project-name: cdk-pattern-test
+        # buildspec-override: path/to/buildspec.yaml or inline buildspec definition
```

Comment:
Lets remove these comments?

Reply:
they are not part of the execution but if you ever need to override the buildspec for codebuild this will be handy.